src/hotspot/share/opto/vectornode.cpp

```diff
@@ -1042,10 +1042,67 @@ Node* VectorNode::try_to_gen_masked_vector(PhaseGVN* gvn, Node* node, const Type
   }
 }
 
+bool VectorNode::can_swap_inputs() {
```

Comment:
```suggestion
bool VectorNode::should_swap_inputs() {
```

I think the method name `can_swap_inputs` should rather be `should_swap_inputs`. Because we `can` swap the edges even when the order is `in(1)->_idx <= in(2)->_idx` already, but we `should` not ;)